plugin/web/extensions/sidebar/core/components/desktop/hooks/projectHooks.ts

```diff
@@ -54,6 +54,14 @@ export const defaultProject: Project = {
       },
     ],
     atmosphere: { shadows: true },
+    light: {
```

Comment:
This also needs to be updated in the mobile's projectHooks file too (sorry due to a lack of time I didn't consolidate shared hooks yet between desktop and mobile...)

Reply:
Updated!